src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(result, number) {
+      result += number;
```

Comment:
The 'add' method should update the 'result' property of the 'calculate' object, not the local 'result' parameter. Use 'this.result' to refer to the object property.